src/metabase/util/schema.clj

```diff
@@ -175,3 +175,8 @@
   "Schema for a valid map of embedding params."
   (with-api-error-message (s/maybe {s/Keyword (s/enum "disabled" "enabled" "locked")})
     (tru "value must be a valid embedding params map.")))
+
+(def Any
+  "Same as schema.core/Any, but includes a message"
+  (with-api-error-message s/Any
+    (tru "value must be specified")))
```

Comment:
This message is wrong because `s/Any` accepts `nil`. 

Also if you want to add an API error message to an existing Schema class you do *not* need to define a new schema. Just stick an entry for `schema.core.AnythingSchema` here: https://github.com/metabase/metabase/blob/master/src/metabase/util/schema.clj#L54

But as previously mentioned we shouldn't even do that because `s/Any` validations don't do anything and go against the whole point of the API endpoint validation stuff in the first place

Reply:
I didn't know that the existing schema stuff was there. I'll remove this anyway, but it's useful to know that's there.